path.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // Path represents a set of points to be thought of as a polyline.
 type Path struct {
```

Comment:
what happens if you do `type Path PointSet`. just rename the type. Might simplify `&Path{&PointSet{}}`